cinch/roles/jenkins_master/templates/enable-kerberos-sso.groovy

```diff
@@ -0,0 +1,21 @@
+def plugin = com.sonymobile.jenkins.plugins.kerberossso.PluginImpl.getInstance()
```

Comment:
Since this code has been ported from previously existing sources we are going to merge this PR as-is and create tasks to improve the ported code in the future as requested in this PR.